modin/numpy/arr.py

```diff
@@ -225,7 +225,7 @@ def __init__(
         # we want to do a simple broadcast where we only consider position, as numpy would, rather
         # than pair columns with the same name and multiply them.)
         self._query_compiler = self._query_compiler.reset_index(drop=True)
-        self._query_compiler.columns = range(len(self._query_compiler.columns))
+        self._query_compiler.columns = list(range(len(self._query_compiler.columns)))
```

Comment:
why cannot we use `pandas.RangeIndex` here?

Reply:
We're just going to use a range, since its the easiest!